wiremock/stubs/referrals.json

```diff
@@ -4,7 +4,6 @@
     "additionalInformation": "",
     "oasysConfirmed": false,
     "offeringId": "e78237cd-8244-4256-a1e8-8fb29b1f2f76",
-    "prisonNumber": "DEF1234",
-    "referrerId": "5105a589-75b3-4ca0-9433-b96228c1c8f3"
+    "prisonNumber": "DEF1234"
```

Comment:
Do we need to put `referrerUsername` on here?

Reply:
Yeah and the other missing `Referral` props too, will add.